code/_globalvars/misc.dm

```diff
@@ -20,3 +20,7 @@ var/map_ready = 0
 	However, using lots of spawn() has a severe performance impact, and often results in spaghetti-code
 	map_ready will be set to 1 when world/New() is called (which happens just after the map is instanced)
 */
+
+var/global/current_year = null
+var/global/current_month = null
+var/global/current_day = null

```

Comment:
I'm guessing the line-of-thought was to prevent having to further do this:

```C++
current_year = text2num(time2text(world.realtime, "YYYY"))
```

Though currently it doesn't if you don't want error prone code, for the reason I gave above.